### DISCUSSION TITLE
Clusters

### DISCUSSION THREAD

**Discussion by R3DD3VIL1982** [Ch4r0ne/ARK-Ascended-Server-Manager#49] [General]
Hi is there a way i can link 2 servers and have both running at once. Thanks

**Comment by dphresh** [comment]
<img width="952" height="352" alt="image" src="https://github.com/user-attachments/assets/dbba04fb-76bf-47de-85c4-e20b1bb8152f" />
This is working for me.  At first I had a custom path set and didn't check "Enable Cluster Custom Path" and that did not work.
